scripts/ci-build.sh

```diff
@@ -0,0 +1,512 @@
+#!/bin/bash -e
+
+# (C) 2018 GENIVI Alliance
+# LICENSE: MPLv2
+
+# ---- General settings ----
+
+# Fixed cache location on all agents that support this.
+AGENT_STANDARD_DL_DIR="/var/cache/yocto/downloads"
+AGENT_STANDARD_SSTATE_DIR="/var/cache/yocto/sstate"
+AGENT_STANDARD_SGX_LOCATION="/var/go/sgx_bin"
+AGENT_STANDARD_SGX_GEN3_LOCATION="/var/go/sgx_bin_gen3"
+
+# ---- Helper functions ----
+
+fail=false
+
+# This is some trickery, but quite useful.
+# Return the value of a variable whose name is stored in another variable
+deref() { eval echo \$$1 ; }
```

Comment:
Why not just call ${other_variable}?

Reply:
${var} returns the value stored in var
deref $var returns the value stored in the variable whose name is stored in var
e.g.
x=y
y=z
echo $x <- prints y
echo $(deref $x) <- prints z

Reply:
also
```
x=y
y=z
echo ${$x}  <- you'd think this should work, but...
--> bash: ${$x}: bad substitution
```
But maybe there's another way...